src/main/java/racing/InputView.java

```diff
@@ -0,0 +1,20 @@
+package racing;
+
+import java.util.Scanner;
+
+public class InputView {
+    private static final Scanner scanner = new Scanner(System.in);
+    public static int getCarCount() {
```

Comment:
가독성을 위해 적절한 개행을 추가해 보시면 좋을 것 같아요
```suggestion
public class InputView {

    private static final Scanner scanner = new Scanner(System.in);
    
    public static int getCarCount() {
```